valhalla/thor/timedistancematrix.h

```diff
@@ -120,8 +122,6 @@ class TimeDistanceMatrix {
    * Reset all origin-specific information
    */
   inline void reset() {
-    // Clear the edge labels and destination list
-    edgelabels_.clear();
```

Comment:
review this again, maybe we want to clear the labels?

Reply:
this is being called at the end of every for loop where we expand from each origin. so we want to clear here, but we also need to reserve some space in the beginning of that for loop in the cc file.